components/tikv_util/src/metrics/mod.rs

```diff
@@ -49,6 +49,7 @@ pub fn dump_to(w: &mut impl Write, should_simplify: bool) {
         if let Err(e) = encoder.encode(&*metric_families, w) {
             warn!("prometheus encoding error"; "err" => ?e);
         }
+        return;
```

Comment:
Please add test case to cover this code path.

Reply:
done